src/main/java/training/subsystems/Arm/Elbow/NeoElbow/NeoElbow.java

```diff
@@ -0,0 +1,65 @@
+package training.subsystems.Arm.Elbow.NeoElbow;
+
+import com.revrobotics.CANSparkBase;
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.math.geometry.Rotation2d;
+import training.subsystems.Arm.Elbow.ElbowConstants;
+import training.subsystems.Arm.Elbow.ElbowInputsAutoLogged;
+import training.subsystems.Arm.Elbow.IElbow;
+
+public class NeoElbow implements IElbow {
+
+	private final CANSparkMax motor;
+
+	public NeoElbow() {
+		this.motor = new CANSparkMax(NeoElbowConstants.MOTOR_ID, CANSparkLowLevel.MotorType.kBrushless);
+		motor.getEncoder().setPositionConversionFactor(ElbowConstants.GEAR_RATIO);
+		motor.getEncoder().setVelocityConversionFactor(ElbowConstants.GEAR_RATIO);
+		motor.getPIDController().setP(NeoElbowConstants.P);
+		motor.getPIDController().setI(NeoElbowConstants.I);
+		motor.getPIDController().setD(NeoElbowConstants.D);
```

Comment:
check about the burn flush function, i think its needed

Reply:
cant fine it, link?

Reply:
mootor.burnFlush()

Reply:
i found burnFlash(), "Writes all settings to flash."
whats the flash?

Reply:
read docs or google it i am not sure